### PR TITLE
Add the tab bar with tab names

### PR DESCRIPTION
**Pull request by sultan-duisenbay**
Add the tab bar (top of the screenshot) - 
Screenshot 2023-04-20 at 5 05 41 PM

### Notable changes
- Added a tab bar with tab names based on mwc components (`mwc-tab` & `mwc-tab-bar`)
- Added the test to make sure tab bar is rendered correctly

### CODE REVIEW

File: web/test/dashboard-summary.test.ts [hunk 1]
```diff
@@ -39,4 +39,19 @@ describe('Testgrid Dashboard Summary page', () => {
...
+    await waitUntil(
+      () => element.shadowRoot!.querySelector('mwc-tab'),
+      'Dashboard summary did not render tabs within the tab bar',
+      {
+        timeout: 4000,
+      },
+    );
+
+    expect(element.tabNames).to.not.be.empty;
+    expect(element.tabNames.length).to.not.equal(1);
```
Comment (chases2) [inline web/test/dashboard-summary.test.ts:55]: What data are we using to run these unit tests? For a test, we should know how many tab names there are and assert that number.

**Review by chases2 — COMMENTED** [review]
Looks good overall, but I have one question about how we're unit testing these components.

File: web/test/dashboard-summary.test.ts (hunk 1 above)
Comment (chases2) [reply web/test/dashboard-summary.test.ts:55]: Please add that as a TODO, either in the code or in a bug. "Creating a mock API" is a substantial, parallel task.

Comment (sultan-duisenbay) [reply web/test/dashboard-summary.test.ts:55]: Got it , thanks!

**Pull request merged**